core/shared/utils/bh_common.h

```diff
@@ -19,6 +19,13 @@ extern "C" {
         bh_assert(_ret == 0);                                         \
     } while (0)
 
+#define bh_memcpy_aw(dest, dlen, src, slen)                           \
```

Comment:
Not sure what `aw` means? Could you add comments?

Reply:
aw - Align Word